tests/core/test_log.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import pytest
+
+from pytest_ethereum.testing import Log
+
+
+logging.getLogger("evm").setLevel(logging.INFO)
+
+
+@pytest.fixture
+def ping(vyper_project_dir, vy_deployer):
+    return vy_deployer.deploy("ping")
+
+
+def test_log_is_present(ping, w3):
+    ping_instance = ping.deployments.get_contract_instance("ping")
+    tx_hash = ping_instance.functions.ping(b"1", b"2").transact()
+    receipt = w3.eth.waitForTransactionReceipt(tx_hash)
+    assert Log(ping_instance.events.Ping, b"1", b"2").is_present(receipt)
+    assert Log(ping_instance.events.Ping, b"1").is_present(receipt)
```

Comment:
So, args checks if any arg is set to this value, but kwargs (from below testcase) must be exact matches?

Clarify with a comment :)